.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Mark stale issues and pull requests
```

Comment:
This is a direct copy from: https://github.com/azavea/docker-django/blob/master/.github/workflows/stale.yml.

Reply:
Seems fine as a first pass especially for what's presumably a low volume repo. Can always tweak the frequency here later.